examples/file/stream/Stream.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using NationalInstruments.SystemLink.Clients.File;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.File
+{
+    /// <summary>
+    /// Example for the SystemLink File API that uploads a file to the
+    /// SystemLink server using an in-memory stream.
+    /// </summary>
+    class Stream
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
+
+            // Use the FileUploader for communicating with the server.
+            var fileUploader = new FileUploader(configuration);
+
+            var fileName = "stream.txt";
+            var fileContents = Encoding.UTF8.GetBytes("ABCDEFGHIJKLMNOPQRSTUVWXYZ");
+
+            // Use in-memory data for the file upload.
+            using (var memoryStream = new MemoryStream())
```

Comment:
So there's something different between this example and the test monitor one, do you think we should show uploading a file from disk instead? I believe we can have the csproj file reference FileViewer.png as a copy to output and then we could upload that?

Reply:
I was planning on creating a separate example that does a file upload from disk.